packages/start-plugin-core/tests/importProtection/virtualModules.test.ts

```
@@ -0,0 +1,240 @@
+import { describe, expect, test } from 'vitest'
+import {
+  MOCK_MODULE_ID,
+  MOCK_EDGE_PREFIX,
```

ESLint

[error] Member 'MOCK_EDGE_PREFIX' of the import declaration should be sorted alphabetically.

(sort-imports)